typescript/sdk/src/core/HyperlaneCore.ts

```diff
@@ -61,10 +61,13 @@ export class HyperlaneCore extends HyperlaneApp<CoreFactories> {
     owners: Address | ChainMap<OwnableConfig>,
   ): ChainMap<RouterConfig> => {
     // get config
-    const config = objMap(this.contractsMap, (chain, contracts) => ({
-      mailbox: contracts.mailbox.address,
-      owner: typeof owners === 'string' ? owners : owners[chain].owner,
-    }));
+    const config = objMap(
+      this.contractsMap,
+      (chain, contracts): RouterConfig => ({
+        mailbox: contracts.mailbox.address,
+        owner: typeof owners === 'string' ? owners : owners[chain].owner,
+      }),
+    );
```

Comment:
this is unchanged?

Reply:
Yeah, only added the `RouterConfig` return type because it was complaining